wandb/board/ui/src/components/Views.js

```diff
@@ -94,18 +100,16 @@ class Views extends React.Component {
           icon={this.state.editMode ? 'unhide' : 'configure'}
           onClick={() => this.setState({editMode: !this.state.editMode})}
         />
-        {(!this.props.blank || this.state.editMode) && (
-          <Tab
-            panes={panes}
-            activeIndex={activeIndex}
-            onTabChange={(event, {activeIndex}) => {
-              this.props.setActiveView(
-                this.props.viewType,
-                this.props.tabs[activeIndex] || activeIndex,
-              );
-            }}
-          />
-        )}
+        <Tab
+          panes={panes}
+          activeIndex={activeIndex}
+          onTabChange={(event, {activeIndex}) => {
+            this.props.setActiveView(
+              this.props.viewType,
+              this.props.tabs[activeIndex],
```

Comment:
The  `|| activeIndex,` is something I just added which ensures the most recently opened tab is selected.

Reply:
Missed that when merging, sorry. Will put it back.